src/nupic/utils/SdrMetrics.hpp

```diff
@@ -188,30 +188,30 @@ class ActivationFrequency : public MetricsHelper_ {
      * instance by assigning to this SDR.
      *
      * @param period Time scale for exponential moving average.
```

Comment:
OT, but please explain time-scale even a bit more

Reply:
Would it help if I changed it from "time scale" to "time constant"?  Since the later appears to be the correct terminology and it even has its own wiki page: https://en.wikipedia.org/wiki/Time_constant